run.py

```diff
@@ -0,0 +1,14 @@
+#!flask/bin/python
+import sys
+from app import mulungwishi_app
+
+# This line is important so flake8 must ignore this one. 
```

Comment:
your comment needs to answer these questions:

1. which error are you silencing?
2. why are you silencing it

“it's important” answers neither :)